### PR TITLE
Rework SQL support section

### PR DESCRIPTION
**Pull request by Jessie212**
## Description

- Removed duplicate content and rearranged SQL section.

## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

**Review by mosabua — COMMENTED** [review]
The "Updating data" and "Data management" sections are duplicate content we should consolidate under "Data management".

File: docs/src/main/sphinx/connector/delta-lake.md [hunk 1]
```diff
@@ -567,19 +498,84 @@ Write operations are supported for tables stored on the following systems:
...
 
-- {doc}`/sql/create-schema`
-- {doc}`/sql/drop-schema`
-- {doc}`/sql/alter-schema`
-- {doc}`/sql/create-table`
+- {doc}`/sql/create-table`, see also {ref}`delta-lake-sql-basic-usage`
 - {doc}`/sql/create-table-as`
 - {doc}`/sql/drop-table`
-- {doc}`/sql/alter-table`
+- {doc}`/sql/alter-table` , see also {ref}`delta-lake-alter-table`
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/delta-lake.md:510]: there are two section .. alter table execute and alter table rename .. maybe consolidate as one with subesctions

File: docs/src/main/sphinx/connector/delta-lake.md [hunk 2]
```diff
@@ -567,19 +498,84 @@ Write operations are supported for tables stored on the following systems:
...
+- {doc}`/sql/create-schema`, see also {ref}`delta-lake-sql-basic-usage`
+- {doc}`/sql/drop-schema`
+- {doc}`/sql/alter-schema`
 - {doc}`/sql/comment`
 
+(delta-lake-sql-basic-usage)=
+
+#### Basic usage examples
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/delta-lake.md:518]: This seems to be all about table and schema management .. move it there .. the title is misleading anyway .. nothing basic about creating schemas and tables..

**Review by mosabua — CHANGES_REQUESTED** [review]
A bit more rejigging needed

File: docs/src/main/sphinx/connector/delta-lake.md [hunk 3]
```diff
@@ -618,7 +590,7 @@ WHERE partition_key = 1
 
 (delta-lake-alter-table)=
 
-#### ALTER TABLE RENAME TO
+##### ALTER TABLE RENAME TO
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/delta-lake.md:593]: This should stay same level as ALTER TABLE EXECUTE

**Review by mosabua — CHANGES_REQUESTED** [review]

File: docs/src/main/sphinx/connector/delta-lake.md (hunk 3 above)
Comment (mosabua) [reply docs/src/main/sphinx/connector/delta-lake.md:593]: And be linked to in the above list

**Comment by Jessie212** [comment]
@mosabua Does everything in this section makes sense in this order?

**Comment by mosabua** [comment]
> @mosabua Does everything in this section makes sense in this order?

Kind of .. proecedures is a bit odd and could be later .. but lets leave it for now

**Review by mosabua — APPROVED** [review]
Looks great now.

**Pull request merged**